unblob/processing.py

```diff
@@ -158,13 +159,21 @@ def _process_regular_file(self, task: Task, size: int, result: TaskResult):
                     )
                     continue
 
-                new_dir = extract_with_command(
-                    extract_dir, carved_valid_path, chunk.handler, result
-                )
+                inpath, outdir = get_extract_paths(extract_dir, carved_valid_path)
+                command = chunk.handler.make_extract_command(str(inpath), str(outdir))
+
+                if not command:
+                    logger.debug(
+                        "No need to extract, as this handler does not have extractor",
+                        handler=chunk.handler.NAME,
```

Comment:
I would increase the verbosity of this log to 2.

Reply:
The verbosity thingy is in a pull request right now, not part of the main branch - but I like the way you are so familiar with it early on! :smile: 

I leave this conversation unresolved and either fix it here if the verbosity stuff merged first, or fix it there if this is merged first.

Reply:
Let's do it once verbosity branch gets merged :)